src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let sum = initialValue;
+    let start = 0;
```

Comment:
What does it mean `start` ? Your variable names should be meaningful. Better rename to `startIndex`.